src/hueClient.js

```diff
@@ -0,0 +1,52 @@
+const fetch = require("node-fetch")
```

Comment:
Can be removed for the reasons I stated in [this comment](https://github.com/bradgarropy/hue-sdk/pull/5#issuecomment-769566365).